admin/scss/bootstrap/_variables.scss

```diff
@@ -135,7 +137,7 @@ $body-color:                 $gray-dark;
 // Set the number of columns and specify the width of the gutters.
 
 // $grid-columns:               12;
-// $grid-gutter-width:          1.875rem; // 30px
+$grid-gutter-width:          2.4615rem; // now 32px, was 1.875rem; ~ 30px
```

Comment:
I've kept $spacer in rems with explanation and gutter width is 2x$spacer, I think that should be good.